src/services/userService.ts

```diff
@@ -0,0 +1,44 @@
+import { User } from './../db/schema/user'
+import bcrypt from 'bcrypt'
+import { UserDocument } from './../utils/types/userType'
+import jwt from 'jsonwebtoken'
+
+export class UserService {
+    static async tryToGetUser(email: string): Promise<UserDocument[]> {
```

Comment:
Is there any reason to return an array of users instead of the first result? You can change `.find` to `.findOne` and `UserDocument[]` to `UserDocument | null`, then in the call site check if result is null instead of checking `.length`